hack/build/bazel-docker.sh

```diff
@@ -38,40 +38,36 @@ EOF
 fi
 
 # Create the persistent docker volume
-if [ -z "$(docker volume list | grep ${BUILDER_VOLUME})" ]; then
-    if [ "$KUBEVIRTCI_RUNTIME" = "podman" ]; then
-        docker volume create ${BUILDER_VOLUME}
-    else
-        docker volume create --name ${BUILDER_VOLUME}
-    fi
+if [ -z "$(${CDI_CRI} volume list | grep ${BUILDER_VOLUME})" ]; then
+    ${CDI_CRI} volume create ${BUILDER_VOLUME}
 fi
 
 # Make sure that the output directory exists
 echo "Making sure output directory exists..."
-if [ "$KUBEVIRTCI_RUNTIME" = "podman" ]; then
-    docker run -v "${BUILDER_VOLUME}:/root:rw,z" --security-opt label=disable --rm --entrypoint "/entrypoint-bazel.sh" ${BUILDER_IMAGE} mkdir -p /root/go/src/kubevirt.io/containerized-data-importer/_out
+if [ "$CDI_CRI" = "docker" ]; then
+    ${CDI_CRI} run -v "${BUILDER_VOLUME}:/root:rw,z" --security-opt label:disable --rm --entrypoint "/entrypoint-bazel.sh" ${BUILDER_IMAGE} mkdir -p /root/go/src/kubevirt.io/containerized-data-importer/_out/gocache
 else
-    docker run -v "${BUILDER_VOLUME}:/root:rw,z" --security-opt label:disable --rm --entrypoint "/entrypoint-bazel.sh" ${BUILDER_IMAGE} mkdir -p /root/go/src/kubevirt.io/containerized-data-importer/_out
+    ${CDI_CRI} run -v "${BUILDER_VOLUME}:/root:rw,z" --security-opt label=disable --rm --entrypoint "/entrypoint-bazel.sh" ${BUILDER_IMAGE} mkdir -p /root/go/src/kubevirt.io/containerized-data-importer/_out/gocache
```

Comment:
I think that `=` worked for both docker and podman
(appears in several places in the PR)